frontend/src/pages/NewRun.test.tsx

```diff
@@ -38,19 +37,23 @@ class TestNewRun extends NewRun {
 }
 
 describe('NewRun', () => {
+  const consoleErrorSpy = jest.spyOn(console, 'error');
```

Comment:
Now looking at this, I'm wondering how this works. I can see the same thing appears in other suites, but if we're not resetting the spy, how come spying on a spy doesn't fail?

Reply:
Aren't I resetting it below in the `beforeEach` ?

Reply:
Yes, but that's `beforeEach`, which means when this suite is done, the method will be mocked. I'm not sure what's happening here, perhaps it's getting reset automatically or something.